### DISCUSSION TITLE
Change back to systems default theme in nested data-theme component?

### DISCUSSION THREAD

**Discussion by TimerErTim** [saadeghi/daisyui#1547] [Q&A] [answered]
Basically, I want to switch back to the system dependent default theme inside a `<body data-theme="dark">` tag. I want this to use as a preview for the user switching themes (including the systems default ones). Is there any way to achieve this?

Code example:

```jsx
<body data-theme="dark"> // Has dark background
    <div data-theme="systems-default" className="btn"> // Should display text and button background according to systems theme
        Test button
    </div>
</body>
```

**Comment by saadeghi** [comment, marked as answer]
The default dark/light theme is set for the `:root` element so there's no built-in feature to set the system's default only for a specific element yet.  
But it is possible by defining a new theme called `systems-default` in the CSS with both dark and light colors and then use it with `data-theme`.
Example: https://play.tailwindcss.com/NvUtQEeB2B?file=css

**Comment by TimerErTim** [comment]
Hey, thanks for the answer.

I ended up doing a manual JavaScript media query check to determine the
systems dark mode and then apply the according data-theme. This was pretty
straightforward since I am using React and it allows me to use already
existing themes.
